cmd/flag.go

```diff
@@ -20,6 +22,8 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&f.Organization, "organization", "o", "giantswarm", "organization to tag images for")
 	cmd.Flags().StringVarP(&f.Username, "username", "u", "", "username to authenticate against registry")
 	cmd.Flags().StringVarP(&f.Password, "password", "p", "", "password to authenticate against registry")
+	cmd.Flags().StringVarP(&f.AccessKey, "access-key", "", "", "access key for registry api")
```

Comment:
Put these beauties up top to keep the alphabetical order.

Reply:
Done